subtests/docker_cli/systemd/systemd_run.py

```diff
@@ -0,0 +1,66 @@
+"""
+Summary
+-------
+Testing basic container as systemd service
+
+Operational Summary
+---------------------
+This is to test container as systemd service using a predefined unit file
+(``p4321.service``) and a ``script p4321-server.py``. An image is built with
+this script added. Systemd starts a container using this image & the container
+writes current time to port ``4321``. From host, the socket is read and the
+value is checked to see if it is a digit. Finally, container is stopped
+through systemd and system is cleaned up.
+
+"""
+
+import time
+import socket
+from autotest.client import utils
+from autotest.client.shared.utils import is_port_free
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import mustpass
+from dockertest.xceptions import DockerTestError
+from systemd import systemd_base
+
+
+class systemd_run(systemd_base):
+
+    def initialize(self):
+        super(systemd_run, self).initialize()
+        dkrcmd = DockerCmd(self, 'build', ['--force-rm -t',
+                                           self.config['image_name'],
+                                           self.sub_stuff['bindir']])
+        mustpass(dkrcmd.execute())
+
+    def run_once(self):
+        host = 'localhost'
+        port = 4321
+        super(systemd_run, self).sysd_action('daemon-reload')
+        super(systemd_run, self).sysd_action('start',
+                                             self.config['sysd_unit_file'])
```

Comment:
No need for ``super()`` here, that's only necessary for "special" methods (like ``__init__()`` or ``run_once()``) or where you must have a guarantee of where in the class ancestry it's looked up (maybe it's called from a method with the same name). For regular-ole-methods, just pretend as if they're already defined in the class.  So, ``self.sysd_action(...)`` in this case.